Dependencies/Utility/Utility/time_compat.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
```

Comment:
While at it, can you add GPL text headers to all these files? Just copy them from Zero Hour source files and change the Copyright holder.

You can also add that to the other files in this Utility folder.